lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,20 +334,31 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
-          apipie :method, "Returns content of 'SSL CA file' configured in Settings > Authentication" do
+          apipie :method, "Returns a combination of 'Server CA file' and 'SSL CA file' configured in Settings > Authentication" do
             example "SSL_CA_CERT=$(mktemp)
                      cat > $SSL_CA_CERT <<CA_CONTENT
                      <%= foreman_server_ca_cert %>
                      CA_CONTENT
-                     curl --cacert $SSL_CA_CERT https://smart-proxy.example.com:8443"
-          end
-          def foreman_server_ca_cert
-            if File.exist?(Setting[:ssl_ca_file])
-              File.read(Setting[:ssl_ca_file])
-            else
-              msg = N_("SSL CA file not found, check the 'SSL CA file' in Settings > Authentication")
-              raise Foreman::Exception.new(msg)
+                     curl --cacert $SSL_CA_CERT https://foreman.example.com:8443"
+          end
+          def foreman_server_ca_cert(server_ca_file_enabled: true, ssl_ca_file_enabled: true)
+            setting_values = []
+            setting_values << Setting[:server_ca_file] if server_ca_file_enabled
+            setting_values << Setting[:ssl_ca_file] if ssl_ca_file_enabled
+            files_content = setting_values.compact.map do |setting_value|
```

Comment:
if both are set, we get both the certificates, right? Is that what we want?

Reply:
this is how I understood this comment https://github.com/theforeman/foreman/pull/8076#issuecomment-717133702 🙂

Reply:
got it :) I don't know enough to judge, just seemed weird to me as they are most likely the same cert :)